Pyjion/pyjit.cpp

```diff
@@ -517,53 +564,136 @@ void PyjionJitFree(PyjionJittedCode* function) {
         delete code;
         g_pyjionJittedCode.erase(function);
     }
-#else
-    delete (TraceInfo*)function->j_evalstate;
 #endif
+	delete obj;
+}
+
+static PyObject *pyjion_enable(PyObject *self, PyObject* args) {
+	auto ts = PyThreadState_Get();
+	auto prev = ts->interp->eval_frame;
+	ts->interp->eval_frame = PyJit_EvalFrame;
+	return prev == PyJit_EvalFrame ? Py_False : Py_True;
+}
+
+static PyObject *pyjion_disable(PyObject *self, PyObject* args) {
+	auto ts = PyThreadState_Get();
+	auto prev = ts->interp->eval_frame;
+	ts->interp->eval_frame = _PyEval_EvalFrameDefault;
+	return prev == PyJit_EvalFrame ? Py_True : Py_False;
```

Comment:
Use `Py_RETURN_TRUE`/`Py_RETURN_FALSE` to increment appropriately.

https://docs.python.org/3.6/c-api/bool.html#c.Py_RETURN_FALSE